resources/migrations/000_migrations.yaml

```diff
@@ -8340,6 +8340,20 @@ databaseChangeLog:
                                   'DateTimeField', 'DecimalField', 'DictionaryField', 'FloatField', 'IntegerField',
                                   'TextField', 'TimeField', 'UUIDField', 'UnknownField');
 
+  - changeSet:
+      id: 307
+      author: dpsutton
+      comment: 'Added 0.40.0 - Add type to collections'
+      changes:
+        - addColumn:
+            tableName: collection
+            columns:
+              - column:
+                  name: type
+                  type: varchar(255)
+                  remarks: 'Nullable column to incidate collection type'
```

Comment:
People can get this information without this comment, it would be a little nicer if the comment explained _what_ a Collection type or something about the purpose of the column